schemas/json/aas.json

```diff
@@ -484,7 +484,7 @@
         },
         {
           "properties": {
-            "value": {
+            "values": {
```

Comment:
there was a reason to have value without "s" for all submodelElements

Reply:
Ok, I'll revert it.

Reply:
On a second thought, are you sure? All the other properties are pluralized, also in RDF. Do you happen to remember who introduced that?